src/Common/Configuration/Properties/Resources.Designer.cs

```diff
@@ -39,7 +39,7 @@ internal class Resources {
         internal static global::System.Resources.ResourceManager ResourceManager {
             get {
                 if (object.ReferenceEquals(resourceMan, null)) {
-                    global::System.Resources.ResourceManager temp = new global::System.Resources.ResourceManager("Microsoft.WindowsAzure.Resources", typeof(Resources).Assembly);
+                    global::System.Resources.ResourceManager temp = new global::System.Resources.ResourceManager("Microsoft.WindowsAzure.Properties.Resources", typeof(Resources).Assembly);
```

Comment:
Absolutely, I have just rolled back these changes. Just a side note: as the resources designer is generated code, and as the namespace tends to follow project structure, any changes to the resources file will automatically result in this namespace change in the future.  Thanks again for your continued support for this new feature.